src/Components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../Todo/TodoItem';
+
+type Props = {
+  todos: Todo[];
+  onTodoRemove: (todoId: number) => void;
+  isRemCompleted: boolean;
```

Comment:
It's better to use a more descriptive name for the 'isRemCompleted' prop, such as 'isRemovingCompleted'.